WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -486,6 +491,26 @@ public static boolean fetchMediaAndDoNext(Context context, Uri mediaUri, MediaFe
         }
     }
 
+    public static List<Uri> retrieveImageEditorResultAndTrackEvent(Intent data) {
+        if (data != null && data.hasExtra(PreviewImageFragment.ARG_EDIT_IMAGE_DATA)) {
+            Map<String, String> properties = new HashMap<>();
+            properties.put("actions", "crop");
+            AnalyticsTracker.track(Stat.MEDIA_EDITOR_USED, properties);
```

Comment:
I'm not sure about having the track event here. However, I wanted to re-use the same code in EditPostActivity.onActivityResult and PhotoPickerActivity.onActivityResult and I couldn't think of a better place. Any ideas?

Reply:
We can move it to a better place when we start tracking edit action for individual images. I compared with iOS implementation, it too doesn't capture individual image edit action. So we're good for now. It's nice that you've included "track" in the method name.